galsim/interpolatedimage.py

```diff
@@ -435,6 +435,10 @@ def __init__(self, image, x_interpolant=None, k_interpolant=None, normalization=
                 pad_image.image, self.x_interpolant, self.k_interpolant, pad_factor,
                 _force_stepk, _force_maxk, gsparams)
 
+        if (calculate_stepk or calculate_maxk) and sbii.getFlux() == 0.:
```

Comment:
Why only raise this exception if ```calculate_stepk or calculate_maxk```?  I agree that if those aren't set, it won't raise immediate problems to have an empty surface brightness profile.  But doesn't this just kick the problems further down the line?